### PR TITLE
5-tickets done with TDD

### PR DESCRIPTION
**Pull request by Hassen-Ahmed**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -12,55 +12,99 @@ afterAll(() => {
...
-            });
+    describe("GET /api/articles", () => {
+        test("200: /api/articles :- should return :200 and list of articles with comment_count", () => {
+            return request(app)
+                .get("/api/articles")
+                .expect(200)
+                .then(({ body }) => {
+                    const articles = body.articles;
+
+                    expect(articles).toHaveLength(13); // ???
```
Comment (MKD01) [inline __tests__/app.test.js:77]: Dont forget to remove comments

File: __tests__/app.test.js [hunk 2]
```diff
@@ -12,55 +12,99 @@ afterAll(() => {
...
+                });
+        });
+        test("404: /api/articles/:article_id :- should return :404 and msg of Not found!", () => {
+            return request(app)
+                .get("/api/articles/999")
+                .expect(404)
+                .then(({ body }) => {
+                    expect(body.msg).toBe("Not found!");
+                });
+        });
```
Comment (MKD01) [inline __tests__/app.test.js:108]: These two tests and not appropriate for this describe block as this endpoint is for /api/articles which does not have any parametric endpoints (that would be your previous task with /api/articles/an_article_id which you have checked for with already).

File: models/articles.model.js [hunk 3]
```diff
@@ -8,3 +8,26 @@ exports.selectArticlesById = (article_id) => {
...
+    LEFT JOIN comments ON comments.article_id = articles.article_id 
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC
+    ;
+    `;
+    const queryValue = [];
+
+    return db.query(query, queryValue).then(({ rows }) => {
+        return rows;
+    });
```
Comment (MKD01) [inline models/articles.model.js:32]: Do you need queryValue here for this to work? Should this be here if it is not doing anything?

**Review by MKD01 — APPROVED** [review]
Great work so far, there are a few changes that need to be made but once you have made those im happy for you to merge this in.

**Comment by Hassen-Ahmed** [comment]
Thank you so much. I will fix then all.

**Pull request merged**